apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/implementation/redirect.md

```diff
@@ -0,0 +1,178 @@
+---
+title: 'Redirect'
+description: 'Recipe for handling redirects using xmcloud and vercel.'
+hasSubPageNav: true
+hasInPageNav: false
+area: ['accelerate']
+---
+
+## Problem
+
+The customer has the following requirements:
+
+- R1. A marketer, strategist, or content author would like to create redirects.
+- R2. The Customer has a huge list of redirects after migration to xmcloud.
+
+## Solution for R1 - small number of redirects
+
+The xmcloud and nextjs app already have O.O.T.B. support for [Redirect](https://doc.sitecore.com/xmc/en/users/xm-cloud/redirect-a-url.html) and [Redirect Map](https://doc.sitecore.com/xmc/en/users/xm-cloud/create-a-redirect-map.html).
+Both solutions are quite useful as long as you do not have a large amount of redirects.
+
+## Solution for R2 - large number of redirects
+
+When you have a large number of redirects, you need to use the hosting provider features to handle it in your frontend application.
+
+For example, vercel has a few methods for handling redirects:
+
+- [Serverless functions](https://vercel.com/docs/edge-network/redirects#serverless-functions)
+- [Edge functions](https://vercel.com/docs/edge-network/redirects#edge-functions)
+- [Edge middleware](https://vercel.com/docs/edge-network/redirects#edge-middleware)
+- [Configuration redirects](https://vercel.com/docs/edge-network/redirects#configuration-redirects)
+
+Each of these methods has its own limitation, for example, configuration redirects supports a maximum 1000 redirects.
```

Comment:
Let's add a section here for each type of redirect:

#### Next.config.js Redirects

Outline what they are here and how to configure then. The Next.js docs are here: https://nextjs.org/docs/app/api-reference/next-config-js/redirects

Add a line about the limitation of 1024 redirects on Vercel. We can move Vercel specific options to the `Discussion` section of the recipe.

Anything on Netlify should also be there.

Reply:
@GuitarRich , i have added a small section for Netlify to the solution and two sections in the discussion. It is based on the official documentation, the links are added to the related docs section